frontend/components/AddSite/TemplateSiteList/index.js

```diff
@@ -30,13 +30,13 @@ export class TemplateList extends React.Component {
   render() {
     const { templates } = this.props;
 
-    const templateGrid = Object.keys(templates).map((templateName, index) => {
-      const template = templates[templateName];
+    const templateGrid = Object.keys(templates).map((templateKey, index) => {
+      const template = templates[templateKey];
 
       return (
-        <div className="federalist-template-list" key={templateName}>
+        <div className="federalist-template-list" key={templateKey}>
           <TemplateSite
-            name={template.title}
+            templateKey={templateKey}
```

Comment:
this was the relevant change, @wslack. Basically the `title` of the template (like `Report Template`) was incorrectly being passed here.

Reply:
right; we shouldn't be pulling this from the template data set. I don't see how you are deriving templateKey unless the key is intended to refer to the entire set of fields about each template.

But we can test that in staging.

Reply:
[line 33](https://github.com/18F/federalist/pull/1656/files/765fb86589f255cd62bf52d29a9d1a9b5011bb01#diff-27d321a5b60a996ab69fc77a401e9c3bR33) shows where `templateKey` comes from.

Reply:
cool; I think I get what the code is doing (assigning an identifier to the collection of variables)